Libraries/Notification/NotificationAndroid.js

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
@dmmiller Most of other components don't follow this approach (`ActionSheetIOS`), some do (`AppStateIOS`). But yeah, I think it'll be a good idea to have one as `.ios.js`.